uk.co.compucorp.civicrm.hrleaveandabsences/CRM/HRLeaveAndAbsences/Service/LeaveRequest.php

```diff
@@ -375,4 +383,46 @@ private function skipBalanceChangeUpdate($params) {
 
     return ($leaveRequestDatesNotChanged || $toilDatesAndToAccrueNotChanged) && !$updateBalanceChange;
   }
+
+  /**
+   * Checks If the current user can cancel a TOIL request type that has past dates.
+   *
+   * @param array $params
+   *
+   * @return bool
+   */
+  private function canCanCancelTOILWithPastDates($params) {
```

Comment:
Do you think it's worth having this method?

Reply:
For one, it makes the code cleaner where the function was used. I can swap it with a variable though, not that I think it really matters.

Reply:
Whichever you like, just asking